src/entities/game.ts

```diff
@@ -10,20 +12,74 @@ export class Game {
   replays: Replay[] = []
   winner: 'left' | 'none' | 'right'
   id: number
+  date?: Date
+  mapName?: string
+  player1?: string
```

Comment:
Later we should factor out the player properties to a separate type to properly support arbitrary number of players.

Reply:
Opened #41 to track this.